### DISCUSSION TITLE
Fill OTP code

### DISCUSSION THREAD

**Discussion by x5tr** [Authenticator-Extension/Authenticator#761] [Q&A] [answered]
<You can also suggest feedback by Tweeting with #AuthenticatorFeedback>

**Describe the feature you want:**
Autofill OTP code (or right-clicking textbox and make a button to fill code)

**Why do you want this feature in Authenticator?**
Autofill OTP code will save my time better than Moving mouse to extensions list button

**Comment by Sneezry** [comment, marked as answer]
Autofill is already supported. Open settings menu, select Preferences, you will see autofill option.

![image](https://user-images.githubusercontent.com/1621293/136511997-83e26e64-3689-4acd-865d-69c44e3be5a5.png)

**Comment by bittner** [comment]
Can this question be marked as "answered" / be closed?
